src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegration.java

```diff
@@ -43,6 +43,81 @@
   private static final Pattern LONG_DATE =
       Pattern.compile("^(\\d{4})-(\\d{2})-(\\d{2})T(\\d{2}):(\\d{2}):(\\d{2})Z$");
 
+  private static final Map<String, String> CONTENT_FORMATTER = getContentFormatter();
+
+  private static Map<String, String> getContentFormatter() {
+    Map<String, String> contentFormatter = new LinkedHashMap<>();
+
+    contentFormatter.put("session_id", "sessionId");
+    contentFormatter.put("asset_id", "assetId");
+    contentFormatter.put("pod_id", "podId");
+    contentFormatter.put("total_length", "totalLength");
+    contentFormatter.put("full_episode", "fullEpisode");
+    contentFormatter.put("content_asset_id", "contentAssetId");
+    contentFormatter.put("ad_asset_id", "adAssetId");
+    contentFormatter.put("load_type", "loadType");
+
+    return contentFormatter;
+  }
+
+  private static final Map<String, String> AD_FORMATTER = getAdFormatter();
```

Comment:
nitpick: Could be nice to do `AD_FORMATTER = Collections.unmodifiableMap(getAdFormatter());`

https://stackoverflow.com/a/22636750